src/sentry/static/sentry/app/views/organizationDashboard/index.jsx

```diff
@@ -74,15 +70,7 @@ class Dashboard extends React.Component {
                 orgId={params.orgId}
                 team={team}
                 showBorder={showBorder}
-                title={
```

Comment:
I've watched enough people stumble on this link. Nobody expects to go to settings, and it can be a major void for user flows. IMO we are better off without it until we find something useful.

Reply:
This would be better off in its own PR in case anyone else had strong opinions. fwiw I think it's a reasonable change.

Reply:
Ok I can move it. It is somewhat related to this pull because the header component looks weird as a link.

Reply:
moved

Reply:
Ah sorry I'm a bit late, but if it's semi-related it's okay to just leave it in